internal/uvm/dumpstacks.go

```diff
@@ -0,0 +1,10 @@
+package uvm
+
+import (
+	"context"
+)
+
+func (uvm *UtilityVM) DumpStacks(ctx context.Context) (err error) {
+	uvm.gc.DumpStacks(ctx)
+	return nil
```

Comment:
Why not `return uvm.gc.DumpStacks(ctx)`

Reply:
Also, this might not be a `gc` backed uvm. So we should verify that it is first.

Reply:
How do we verify that?

Reply:
Look at some of the other methods. We test if `gc != nil` otherwise we call a `hcs/*` method. In this case if its `!= linux` and `gc == nil` we just simply skip the method.